_sass/colors/typography-dark.scss

```diff
@@ -104,10 +104,6 @@
     display: none;
   }
 
-  hr {
-    border-color: var(--main-border-color);
-  }
-
```

Comment:
Consider following screenshots. This is how it will appear.

![image](https://github.com/cotes2020/jekyll-theme-chirpy/assets/10548881/b4d71ee0-e58a-4ab4-8ad8-db347404f1db)
![image](https://github.com/cotes2020/jekyll-theme-chirpy/assets/10548881/4745f2aa-c8f1-42a6-ad58-562764f99895)

Reply:
> adjust styles for both light and dark themes then...

That's a good idea, then please add the `hr` code to `_sass/addon/commons.scss`. It is recommended to insert the following code before the `footer` ([line 131](https://github.com/cotes2020/jekyll-theme-chirpy/blob/9592146ca392236e69ee358412ecc32ef1662127/_sass/addon/commons.scss#L131)).


```scss
hr {
  border-color: var(--main-border-color);
  opacity: 1;
}
```

Also remove the `hr` block from `_sass/colors/typography-<dark|light>.scss`

Reply:
Oh, sure. Thanks for the advice.